style.css

```diff
@@ -747,6 +747,17 @@ img:hover {
   transform: scale(1.05);
 }
 
+.error-message.show {
+  color: #fff;
```

Comment:
- [ ] **Optional:** Error message since they signify that something hasn't been done right should be in `color: red`.